migrations/2020_11_14_000000_migrate_settings_to_fof.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of fof/doorman.
+ *
+ * Copyright (c) 2018-2020 Reflar.
+ * Copyright (c) 2020 FriendsOfFlarum
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ *
+ */
+
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        /**
+         * @var \Flarum\Settings\SettingsRepositoryInterface
+         */
+        $settings = app('flarum.settings');
+
+        foreach (['allowPublic'] as $setting) {
+            if ($value = $settings->get($key = "reflar.doorman.$setting")) {
```

Comment:
You could use an SQL query that replaces all strings with `reflar.doorman` with `fof.doorman`?

Reply:
Agreed that it could be done this way, but this is the current president that other FoF extensions use at this time, so I'm inclined to leave this as is for now